app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardVM.kt

```diff
@@ -81,7 +81,7 @@ class DashboardVM @ViewModelInject constructor(
     @OnLifecycleEvent(Lifecycle.Event.ON_START)
     fun onStart(){
         if (!deviceUtils.isBtEnabled() || !deviceUtils.isLocationEnabled()) {
-            navigate(R.id.action_nav_dashboard_to_nav_permission_disabled)
+            safeNavigate(R.id.action_nav_dashboard_to_nav_permission_disabled, R.id.nav_dashboard)
```

Comment:
Proč je potřeba ten druhý parametr? Nejde to zjistit z aktuálního nav grafu?

Reply:
řek bych, že jo -> navController.currentDestination.id

Reply:
Knihovna to neřeší, protože je to problém na úrovní View a ne knihovny.
Když se při double-tap zavolá podruhé `navigate()` zatímco jsi už na jiné obrazovce není chyba knihovny.

`Try-catch` to samozřejmě odchytí a vizuálně to "stačí"